scripts/ci-e2e.sh

```diff
@@ -71,7 +71,7 @@ else
   export CAPM3_FROM_RELEASE="${CAPM3_FROM_RELEASE:-$(get_latest_release "${CAPM3RELEASEPATH}" "v0.5.")}"
   export CAPM3_TO_RELEASE="${CAPM3RELEASE}"
 fi
-
+sudo nft flush ruleset
```

Comment:
why do we need this?

Reply:
could this be one of the commands to fix the broken pipe issue ?
I do not think we need it in this PR.